examples/for-tests/src/App.js

```diff
@@ -168,6 +168,54 @@ const formFields = [
     },
 ];
 
+const customFields = [
+    {
+        id: "ratings",
+        label: "Ratings",
+        inputComponent: ({ value, name, ...rest }) => (
+            <select
+                name={name}
+                data-supertokens="inputComponent"
+                placeholder="Add Ratings"
+                onChange={(e) => rest.onChange({ id: name, value: e.target.value })}>
```

Comment:
understand how onChange works:
- Does it require id and value?
- Can value be anything other than a string?
- If id is required, can be get rid of it?

Reply:
Does it require id and value? -` Yes, both are required`
Can value be anything other than a string? - `No`
If id is required, can be get rid of it? - `Need to write a separate onChange func to handle inputComponent onChange`